Src/main.c

```diff
@@ -315,34 +315,15 @@ uint16_t target_e_com_time_low;
 uint8_t compute_dshot_flag = 0;
 uint8_t crsf_input_channel = 1;
 uint8_t crsf_output_PWM_channel = 2;
-char eeprom_layout_version = 2;
-char dir_reversed = 0;
-char comp_pwm = 1;
-char VARIABLE_PWM = 1;
-char bi_direction = 0;
-char stuck_rotor_protection = 1; // Turn off for Crawlers
-char brake_on_stop = 0;
-char stall_protection = 0;
-char use_sin_start = 0;
-char TLM_ON_INTERVAL = 0;
+char eeprom_layout_version = 3;
 uint8_t telemetry_interval_ms = 30;
-uint8_t TEMPERATURE_LIMIT = 255; // degrees 255 to disable
-char advance_level = 2; // 7.5 degree increments 0 , 7.5, 15, 22.5)
 char temp_advance = 1;
 uint16_t motor_kv = 2000;
```

Comment:
At first I thought this was related to this PR, but this should be renamed. Its not really motor_kv, but really some weird scaler of the motor KV (multiplied by 40 with 20 added after that, then divided by 2 if its a 3s max esc). Regardless this name should be changed as it shares the same name with the eeprom struct member and its easy to mess them up.

Reply:
it actually is motor kv, the code translates this to rpm under which it limits code. The name is correct and the calculation is done to save the value in uint8_t and dont waste memory.

Also totally unrelated to this pr, which is just a refactoring of eeprom data handling for less flash used (saves about 200 - 300b depending on compile configuration)